gr-fosphor.rb

```diff
@@ -17,6 +17,7 @@ def install
       args = %W[
         -DCMAKE_SHARED_LINKER_FLAGS='-Wl,-undefined,dynamic_lookup'
         -DPYTHON_LIBRARY='#{HOMEBREW_PREFIX}/lib/libgnuradio-runtime.dylib'
+        -DFREETYPE2_INCLUDE_DIR_ftheader='#{HOMEBREW_PREFIX}/Cellar/include'
```

Comment:
Are you sure this helps?  There isn't supposed to be an include directory in the cellar.